website/docs/02 - Connection.md

```diff
@@ -96,7 +99,7 @@ MVOM requires a number of database server subroutines (referred to by MVOM as _s
 ### Syntax
```

Comment:
I can't tag the appropriate spot in the file, but I think the description text above is a little misleading now since we only deploy a single subroutine.  I think it just needs some slight tweaking to refer to the `mvom_main` subroutine.